app/views/edit-project.html

```diff
@@ -0,0 +1,43 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+	Edit Project
+{% endblock %}
+
+{% block content %}
+<main id="content" role="main">
+	<div class="grid-row">
+		<section style="padding-top:30px;">
+			<h1 class="form-title heading-large">
+				Edit Project
+			</h1>
+			{% include "includes/project-form.html" %}
+		</section>
+	</div>
+</main>
+{% endblock %}
+
+{% block body_end %}
+{{ super() }}
+<script>
+$("#projectForm").submit(function(e) {
+    e.preventDefault();
+    $.ajax({
+        type: "PUT",
+        url: '/api/projects/edit/{{ project.id }}',
+        contentType: "application/json",
+        data: JSON.stringify($('#projectForm').serializeObject())
+    })
+    .done(function(response) {
+        alert(JSON.stringify(response.messages[0])); 
+		window.location.href = '/';
```

Comment:
lots of tabulation here ;)

EDIT: in general there seems to be an inconsistency tab vs 4 spaces here